health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/SearchFileIds.js

```
@@ -0,0 +1,73 @@
+    }
+    // let userNames = {}
+    console.log("res1", response);
+    console.log("res2", response1);
+    response?.ResourceDetails.forEach((ob) => {
+      ob["username"] = uuidName[ob?.auditDetails?.lastModifiedBy];  // Set the username
+      ob.auditDetails.lastmodtime = formatTimestamp(ob?.auditDetails?.lastModifiedTime);  // Format and set the last modified time
+    });
+
+    console.log("response process1",response)
+
+    let res = {...response}
```

Biome

[error] This let declares a variable that is only assigned once.

'res' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)